### ISSUE TITLE
Build gets stuck at "Building Gradle Project"

### ISSUE THREAD

**Issue by dskill** [oculus-samples/Unity-TheWorldBeyond#3]
Trying to build a .apk and it keeps getting stuck at "Building Gradle Project".  I'm on Windows, Unity 2020.2.34f1. Thx!

**Comment by dskill** [comment]
found this in the logs:

```
CommandInvokationFailure: Gradle build failed. 
C:\Program Files\Unity\Hub\Editor\2020.3.34f1\Editor\Data\PlaybackEngines\AndroidPlayer\OpenJDK\bin\java.exe -classpath "C:\Program Files\Unity\Hub\Editor\2020.3.34f1\Editor\Data\PlaybackEngines\AndroidPlayer\Tools\gradle\lib\gradle-launcher-6.1.1.jar" org.gradle.launcher.GradleMain "-Dorg.gradle.jvmargs=-Xmx4096m" "assembleRelease"

stderr[

]
stdout[
Starting a Gradle Daemon, 1 busy and 1 incompatible Daemons could not be reused, use --status for details
]
```

**Comment by tdmowrer** [comment]
It builds for me on Windows, Unity 2020.3.26f1. This seems like it might be more of a gradle issue generally than a problem with this repo specifically. Are you able to build any other project, e.g., a new project with an empty scene?

**Comment by dskill** [comment]
Ok thx for confirming.  I can build other projects and deploy to my quest without a problem, so whatever it is does seem specific to this repo.  I'll try doing a full system reboot and see if that helps at all.

**Comment by MetaMats** [comment]
I just tested with a fresh clone and Unity 2020.3.34f, and was able to build successfully for Android without any further adjustments. Are you still experiencing this issue? @dskill

**Comment by dskill** [comment]
Unfortunately, still no luck but it sounds like this is unique to my setup somehow.  I'll close the bug for now.  Thx for confirming a fresh clone should have worked fine.

**Comment by ClariceHilton** [comment]
I'm having the same issue on two different computers dskill did you figure out the problem?

**Comment by dskill** [comment]
no, I'm sorry.  I never figured it out. I ended up just using an apk made by a colleague.
